### PR TITLE
feat: waku bloom filter mode switch

### PR DESCRIPTION
**Pull request by richard-ramos**
related to #2877

### CODE REVIEW

File: src/nim_status_client.nim [hunk 1]
```diff
@@ -175,13 +176,12 @@ proc mainProc() =
...
   # do not need a running node
   proc initControllers() =
-    node.init()
     login.init()
     onboarding.init()
 
   initControllers()
 
-  # Handle node.stopped signal when user has logged out
+  # Handle node.stopped signal w.init(hen user has logged out
```
Comment (jrainville) [inline src/nim_status_client.nim:184]: 🤔

**Review by jrainville — APPROVED** [review]

**Comment by iurimatias** [comment]
I will revisit this, currently this branch just straight up crashes for me

**Comment by iurimatias** [comment]
I see the Bloom Mode is off by default, but shouldn't it be true by default? isn't the normal behaviour to have the bloom filter?

**Comment by iurimatias** [comment]
so explanation by Andre on this

```
no, so by default it uses clear topics
if you enable it, it will instead of using clear topics use a bloom filter (so better privacy, worse bandwidth)
there's no way to act as a full node (i.e empty bloom filter) through the mobile client interface (though is possible if we exposed some settings)
```

**Review by iurimatias — APPROVED** [review]

**Pull request merged**